articles/bases.py

```diff
@@ -67,9 +67,8 @@ def get_urls(cls):
     def __str__(self):
         return self.title
 
-    @app_models.permalink
     def get_absolute_url(self):
-        return ('article_detail', 'articles.urls', (), {'slug': self.slug})
```

Comment:
I haven't tested it, but my instinct would suggest no.  It looks like the URLs might be constructed at a funny time.  I'll have a go.